package.json

```diff
@@ -49,16 +50,6 @@
       "git add"
     ]
   },
-  "prettier": {
```

Comment:
Unfortunately we need both configurations: this one is the only one that could be read by tsdx. If you want you can remove the one in .prettier file, but then we will have some problems with VSCode. The configuration that was there, was wisely crafted to work in all scenarios (and it failed anyway :( )

Reply:
This has been added back in.